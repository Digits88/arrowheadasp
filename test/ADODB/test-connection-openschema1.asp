<!--#include file="../config/ADODB.inc"-->
Here is the test of opening a schema.<br>
<%
    Dim CN, RS, adSchemaColumns
    
    adSchemaColumns = 4

    Set CN = Server.CreateObject("ADODB.Connection")
    CN.Open Connect_URL, Connect_Username, Connect_Password
    Set RS = CN.OpenSchema(adSchemaColumns)

    While not RS.Eof
        if (RS("TABLE_NAME") = "Test") then
            %>Column Name: <%=RS("COLUMN_NAME")%><br>
            <%
        end if
        RS.MoveNext
    Wend

    CN.Close
%>

The End.
