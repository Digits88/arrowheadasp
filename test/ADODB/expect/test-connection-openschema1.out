
Here is the test of opening a schema.<br>
Column Name: StringField<br>
            Column Name: IntField<br>
            Column Name: NullableStringField<br>
            Column Name: NullableIntField<br>
            

The End.
